checkov/main.py

```diff
@@ -222,13 +222,19 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
         bc_integration.skip_download = True
 
     bc_integration.get_platform_run_config()
+    bc_integration.get_prisma_build_policies(config.filter)
 
     integration_feature_registry.run_pre_scan()
 
+    if integration_feature_registry.features[0].__dict__.get('filtered_policy_ids'):
```

Comment:
I think it would be better to check if this feature `isinstance(PolicyMetadataIntegration)` and then directly access the `filtered_policy_ids` without the need of using the `__dict__` method.

Reply:
Makes sense